README.md

```diff
@@ -42,10 +42,70 @@ To load data, again with service up run:
 
 ```sh
 docker-compose exec core bash
-python manage.py acquiredata 
+python manage.py acquiredata
 python manage.py loaddata
 ```
 
 To verify, run `curl http://localhost:8000/diseases/` to get a list of all diseases.
 
-Or, run `curl http://localhost:8000/samples?limit=10` to view data for 10 samples.
\ No newline at end of file
+Or, run `curl http://localhost:8000/samples?limit=10` to view data for 10 samples.
+
+## Deployment
+
+### Prerequisites
+
+This project is deployed within the Greene Lab AWS account. To be able
+to deploy this project you will need to:
+1. Be invited to the account.
+2. Receive an AWS access key and secret key.
+
+### Logging Into ECR
+
+This project leverages
+[AWS Ec2 Container Service (ECS)](https://aws.amazon.com/ecs/details).
+ECS provides a private container registry called the
+[Ec2 Container Repository (ECR)](https://aws.amazon.com/ecr/).
+To be able to push Docker images to this repository you will first need to
+get a login with:
+```sh
+aws ecr get-login --region us-east-1
+```
+and then run the output of that command. It will look something like:
+```sh
+docker login -u AWS -p <A_GIANT_HASH> -e none https://934349674827.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
can we use long arguments when available in docs for clarity. i.e. `--password`, `--username`, `--email`